open_lm/train.py

```diff
@@ -132,14 +134,32 @@ def train_one_epoch(model, data, loss, epoch, step, optimizer, scaler, scheduler
 
     end = time.time()
 
+    try:
+        starting_step = int(optimizer.state_dict()["state"][0]["step"].item())
```

Comment:
Can we use the step provided to this function instead of relying on optimizer state? (see also https://github.com/mlfoundations/open_lm/pull/102)

Reply:
My bad, this was an unused artifact from before - I deleted it.